.github/workflows/ci.yml

```diff
@@ -1,18 +1,18 @@
-name: Java CI
```

Comment:
Why these files marked changed and included in your PR - README, checklist, pom.xml, gitignore, ci.yml?